the-9-exercises/src/App.jsx

```diff
@@ -1,14 +1,10 @@
-
-import './App.css'
-
-function App() {
-
-
-  return (
-    <>
-      <h1 className='bg-red-400'>hey there</h1>
-    </>
-  )
+import { v4 as uuidv4 } from "uuid";
+import AnimalList from "./components/AnimalList";
+import data from "./components/data.json";
+
+export default function App() {
+  const ANIMAL = data.map((item) => (
+    <AnimalList id={uuidv4()} name={item.name} image={item.url} />
```

Comment:
It would be good if you add key property when you're mapping through elements to make it possible for react to differentiate your components